### PR TITLE
API interaction

### PR DESCRIPTION
**Pull request by jakedcolbert**

### CODE REVIEW

File: p3/Twentivia/app/src/main/java/com/elijake/twentivia/TriviaActivity.kt [hunk 1]
```diff
@@ -29,9 +29,11 @@ class TriviaActivity : AppCompatActivity() {
         supportActionBar?.hide();
         setContentView(R.layout.activity_questions)
 
+
+
         Thread {
             // network calls need to be done in threads
-            triviaQuestions = TriviaAPI.getTrivia("https://the-trivia-api.com/api/questions?limit=20")
+            triviaQuestions = TriviaAPI.getTrivia("https://the-trivia-api.com/api/questions?limit=5")
```
Comment (elijahbigk77) [inline p3/Twentivia/app/src/main/java/com/elijake/twentivia/TriviaActivity.kt]: why is the question limit set to 5? Is it just for quick testing purposes?

File: p3/Twentivia/app/src/main/java/com/elijake/twentivia/TriviaActivity.kt [hunk 2]
```diff
@@ -115,12 +125,29 @@ class TriviaActivity : AppCompatActivity() {
...
+        }
+    }
 }
 
 
 // Notes for further work:
 // - occasional timer bug where questions 'skip' or timer 'looses time'/'keeps running'
+// - UI changes, specifically purple highlights
 // - after all questions answered, app crashes
 // - no initial "3, 2, 1, go!" for trivia begin, so questions start when user isn't ready
```
Comment (elijahbigk77) [inline p3/Twentivia/app/src/main/java/com/elijake/twentivia/TriviaActivity.kt:149]: good point!

File: p3/Twentivia/app/src/main/java/com/elijake/twentivia/TriviaActivity.kt [hunk 3]
```diff
@@ -115,12 +125,29 @@ class TriviaActivity : AppCompatActivity() {
...
+        for (button in buttons) {
+            button.isClickable = false;
+        }
+    }
 }
 
 
 // Notes for further work:
 // - occasional timer bug where questions 'skip' or timer 'looses time'/'keeps running'
+// - UI changes, specifically purple highlights
```
Comment (elijahbigk77) [inline p3/Twentivia/app/src/main/java/com/elijake/twentivia/TriviaActivity.kt:147]: What do you mean by purple highlights?

**Review by elijahbigk77 — COMMENTED** [review]
Good work!

**Review by elijahbigk77 — COMMENTED** [review]
LGTM!

File: p3/Twentivia/app/src/main/java/com/elijake/twentivia/TriviaActivity.kt (hunk 3 above)
Comment (jakedcolbert) [reply p3/Twentivia/app/src/main/java/com/elijake/twentivia/TriviaActivity.kt:147]: Things like the Top Bar, the purple semicircles that appear when scrolling. Its an easy fix, just changing the theme colors in /values/colors.xml

File: p3/Twentivia/app/src/main/java/com/elijake/twentivia/TriviaActivity.kt (hunk 1 above)
Comment (jakedcolbert) [reply p3/Twentivia/app/src/main/java/com/elijake/twentivia/TriviaActivity.kt]: Yessir, was just for testing. I believe I corrected it with the new API function. Curious why it doesn't appear here

**Pull request merged**